device/tt_cluster_descriptor.h

```diff
@@ -33,11 +33,12 @@ enum BoardType : uint32_t {
 
 class tt_ClusterDescriptor {
 
-  private:
+private:
   int get_ethernet_link_coord_distance(const eth_coord_t &location_a, const eth_coord_t &location_b) const;
 
-  protected:
+protected:
 
+  bool chips_have_coordinates;
```

Comment:
Unused?
Revert all the changes in this file if there are only formatting changes after that...